x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -145,26 +163,100 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
       [severityUpdate, swimLaneSeverity]
     );
 
+    const openCasesModal = useCallback(
+      (swimLaneType: SwimlaneType) => {
+        selectCaseModal!.open({
+          attachments: [
+            {
+              // @ts-ignore
+              type: 'persistableState' as const,
+              persistableStateAttachmentTypeId: ANOMALY_SWIMLANE_EMBEDDABLE_TYPE,
+              persistableStateAttachmentState: {
+                swimlaneType: swimLaneType,
+                ...(swimLaneType === SWIMLANE_TYPE.VIEW_BY
+                  ? { viewBy: viewBySwimlaneFieldName }
+                  : {}),
+                jobIds: selectedJobs?.map((v) => v.id),
+                timeRange: globalTimeRange,
+              } as AnomalySwimlaneEmbeddableCustomInput,
+              owner: PLUGIN_ID,
+            },
+          ],
+        });
+      },
+      [selectCaseModal, selectedJobs, globalTimeRange, viewBySwimlaneFieldName]
+    );
+
     const annotations = useMemo(() => overallAnnotations.annotationsData, [overallAnnotations]);
 
-    const menuItems = useMemo(() => {
-      const items = [];
+    const menuPanels = useMemo<EuiContextMenuPanelDescriptor[]>(() => {
+      const rootItems = [] as EuiContextMenuPanelItemDescriptor[];
+      const panels = [{ id: 0, items: rootItems }] as EuiContextMenuPanelDescriptor[];
```

Comment:
Nit:
```suggestion
      const rootItems: EuiContextMenuPanelItemDescriptor[] = [];
      const panels: EuiContextMenuPanelDescriptor[] = [{ id: 0, items: rootItems }];
```

Reply:
Changed in https://github.com/elastic/kibana/pull/138994/commits/083c565b8e6274052aea7d7934f7396d7d6f57aa